.github/workflows/ci.yml

```diff
@@ -215,20 +215,20 @@ jobs:
           ACCOUNT_INDEX: ${{ steps['determine-test-account'].outputs['account_index'] }}
           ACCESS_KEY_IDS: ${{ secrets.ACCESS_KEY_IDS }}
           SECRET_ACCESS_KEYS: ${{ secrets.SECRET_ACCESS_KEYS }}
-          PSQL_CONN: postgres://postgres:test@localhost:5432/iasql?ssl=true&sslmode=require
+          PSQL_CONN: postgres://postgres:test@localhost:5432/iasql
```

Comment:
why do we remove the ssl mode?

Reply:
So Postgres 14 is able to correctly negotiate the proper SSL level without these query params, unlike Postgres 13, so I was just cleaning things up *but* I now notice that I forgot to make the update to the `pg_hba.conf` files to only allow SSL-encrypted connections, and I also have a merge conflict, so let me rebase and also correct that.

Reply:
oh ok, I was worried about the enforcement of SSL connections, if we are going to update the postgres config to force it I'm ok removing this.